docs/openapi3.yaml

```diff
@@ -2277,3 +2418,38 @@ components:
         percentage:
             type: number
             description: benchmark percentage weight
+    webhook:
+      type: object
+      required:
+        - name
+        - webhook_url
+        - events
+      properties:
```

Comment:
I think it will be clearer add also param called 'format_type' with closed set ['slack', 'json']
also not sure how but we will need to define the API of the requests we will send on the triggers

Reply:
added format_type to webhook schema